torch/csrc/jit/passes/inliner.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include <torch/csrc/jit/ir.h>
+
+namespace torch {
+namespace jit {
+
+TORCH_API void Inline(Block* block, bool recurse = true);
```

Comment:
Can you comment in the code why the recurse flag exists? (not what it is but why it is)